tests/kernel/boot_ltp.pm

```diff
@@ -40,6 +40,11 @@ sub run {
     }
 
     $self->select_serial_terminal;
+
+    # Debug code for poo#81142
+    script_run('cat /dev/fb0 | gzip -9 >framebuffer.dat.gz');
```

Comment:
how about putting that into the post_fail_hook? If you do that as the first action within the post_fail_hook nothing should change what is within the framebuffer

Reply:
Your workaround needle prevents `boot_ltp` failure most of the time so `post_fail_hook()` will not be called.
https://openqa.suse.de/tests/6630340#step/install_ltp/3

This is the most reliable way to gather debug data right now. The gzipped framebuffer file is about 20-30KB in size so the disk overhead is negligible.

Reply:
sure, fine.